obs-script/obs-Auto-League.py

```diff
@@ -454,21 +473,21 @@ def auto_setup(props, prop): # Todo: add BO3 and BO5
         obs.obs_source_release(social_source)
         # obs.obs_sceneitem_release(social_item)
 
-        # Blue-Boost-0
-        temp_settings = get_settings('Blue Boost 0')
-        blue_boost_0_source = obs.obs_source_create('color_source', 'Blue-Boost-0', temp_settings, None)
-        blue_boost_0_item = obs.obs_scene_add(main_scene, blue_boost_0_source)
-        obs.obs_data_release(temp_settings)
-        obs.obs_source_release(blue_boost_0_source)
-        # obs.obs_sceneitem_release(blue_boost_item)
-
-        # Orange-Boost-0
-        temp_settings = get_settings('Orange Boost 0')
-        orange_boost_0_source = obs.obs_source_create('color_source', 'Orange-Boost-0', temp_settings, None)
-        orange_boost_0_item = obs.obs_scene_add(main_scene, orange_boost_0_source)
-        obs.obs_data_release(temp_settings)
-        obs.obs_source_release(orange_boost_0_source)
-        # obs.obs_sceneitem_release(orange_boost_item)
+        # # Blue-Boost-0
+        # temp_settings = get_settings('Blue Boost 0')
+        # blue_boost_0_source = obs.obs_source_create('color_source', 'Blue-Boost-0', temp_settings, None)
+        # blue_boost_0_item = obs.obs_scene_add(main_scene, blue_boost_0_source)
+        # obs.obs_data_release(temp_settings)
+        # obs.obs_source_release(blue_boost_0_source)
+        # # obs.obs_sceneitem_release(blue_boost_item)
+        #
+        # # Orange-Boost-0
+        # temp_settings = get_settings('Orange Boost 0')
+        # orange_boost_0_source = obs.obs_source_create('color_source', 'Orange-Boost-0', temp_settings, None)
+        # orange_boost_0_item = obs.obs_scene_add(main_scene, orange_boost_0_source)
+        # obs.obs_data_release(temp_settings)
+        # obs.obs_source_release(orange_boost_0_source)
+        # # obs.obs_sceneitem_release(orange_boost_item)
```

Comment:
Commiting out-commented code is bad style. Since we are using git you can also go back and fetch it again anyway.

Reply:
I did not change the text size nor did i add developers, could you provide me with text size you used for each?